GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -224,6 +224,14 @@ private IReadOnlyList<MenuCommand> CreateViewMenuCommands()
                     ExecuteAction = () => _revisionGrid.ToggleShowReflogReferences(),
                     IsCheckedFunc = () => AppSettings.ShowReflogReferences
                 },
+                new MenuCommand
+                {
+                    Name = "ShowStashReferences",
+                    Text = "Show stash references",
```

Comment:
This is a single stash, reference**s** implies that all stashes are shown.

Reply:
As before, I do not believe "stash reference" is something that is Git lingo, but "stash" or "stash entry" is maybe not good enough. 
"Show latest stash" or "Show most recent stash"?

Reply:
"Show stash references" was propossed by @RussKie https://github.com/gitextensions/gitextensions/pull/9555#discussion_r701080238 . And I agree with "stash reference" because it is handled by git and gitextensions as a reference named "stash".

But, this is not a hill I would like to die on, ¿What label should I use?